pkg/image/validation/kubernetes.go

```diff
@@ -117,6 +113,48 @@ func validateNodes(k8s *image.Kubernetes) []FailedValidation {
 	return failures
 }
 
+func validateNetwork(k8s *image.Kubernetes) []FailedValidation {
+	var failures []FailedValidation
+	ip := k8s.Network.APIVIP
+
+	numNodes := len(k8s.Nodes)
+	if numNodes <= 1 {
+		// Single node cluster, node configurations are not required
+		return failures
+	}
+
+	if ip == "" {
+		failures = append(failures, FailedValidation{
+			UserMessage: "The 'apiVIP' field is required in the 'network' section when defining entries under 'nodes'.",
+		})
+	}
+
+	if ip != "" {
```

Comment:
Shouldn't all of the validations below also apply for single node clusters? If the value is there, it should be valid.